devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -13,7 +13,7 @@
  */
 
 metadata {
-    definition (name: "ZigBee Switch", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch", runLocally: true, minHubCoreVersion: '000.019.00012', executeCommandsLocally: true) {
+    definition (name: "ZigBee Switch", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch") {
```

Comment:
For your testing you can comment out or remove this code but please leave it as is for the code review